packages/server/src/db/index.ts

```diff
@@ -161,6 +161,9 @@ export class Database {
     }
     record.raw = JSON.stringify(obj)
 
+    record.indexedAt = new Date().toISOString()
+    record.receivedAt = record.indexedAt
```

Comment:
what's the difference here?
is it like "createdAt" vs "updatedAt"?

Reply:
Yeah and so obviously this logic is wrong -- discussing in slack